### DISCUSSION TITLE
Preview of LinkFree forms

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/LinkFree#7770] [Show and tell]
Preview environment https://linkfree-preview.vercel.app

- Please raise any bugs for the `manage-forms` branch in a thread below (each to have a new thread)

**Comment by eddiejaoude** [comment]
BUG: fix icons import (similar to `main` branch)

Example of this being done to `main` branch https://github.com/EddieHubCommunity/LinkFree/pull/7738/files

**Reply by rohan-kulkarni-25** [reply]
Will be working on this !! Do i need to create issue ?

**Reply by Pradumnasaraf** [reply]
Yes, @rohan-kulkarni-25. Create an issue I will assign to you right now.

**Reply by rohan-kulkarni-25** [reply]
This one is solved

**Reply by eddiejaoude** [reply]
Thank you 👍  going to merge now

**Comment by eddiejaoude** [comment]
BUG: light and dark theme to forms (branch `manage-forms`)

**Comment by eddiejaoude** [comment]
BUG: new event / milestone error (I think it is the date input)

**Reply by eddiejaoude** [reply]
FIXED

**Comment by Pradumnasaraf** [comment]
Milestone text and date are getting cut by the pencil icon. We need to shift it a little bit upwards
<img width="1495" alt="Screenshot 2023-06-24 at 6 51 27 PM" src="https://github.com/EddieHubCommunity/LinkFree/assets/51878265/f319eaeb-5526-4847-bf08-457d033049f9">

**Reply by rohan-kulkarni-25** [reply]
We can fix the width of the text box. This will solve the issue it seems. Would like to try if you have not started to work on.

**Comment by Pradumnasaraf** [comment]
The message should be clear. By this statement, it sounds like we can't manage via the form if we have a JSON file. But we can still do that.

Instead, it should say something like, "Once you start using forms, the JSON file does not play any role."

<img width="1495" alt="Screenshot 2023-06-24 at 6 57 41 PM" src="https://github.com/EddieHubCommunity/LinkFree/assets/51878265/2a98b18e-232e-485a-8684-edc409fc95d5">

**Reply by eddiejaoude** [reply]
Thank you 👍  fixed 🚀

**Comment by ikrishg** [comment]
Eddie the domains thing is called [platformize](https://platformize.co/) it is 1st party

**Reply by eddiejaoude** [reply]
Thank you 👍

**Comment by ikrishg** [comment]
without a profile if you login it shows you button "go to forms" which actually creates your profile, but imho the wording is unclear

**Comment by eddiejaoude** [comment]
BUG: social shortcut links still using url as identifier instead of id

**Reply by eddiejaoude** [reply]
FIXED 👍
